code/game/objects/structures/statues.dm

```diff
@@ -293,10 +253,212 @@
 ///////////////////////////////bronze///////////////////////////////////
 
 /obj/structure/statue/bronze
-	material_drop_type = /obj/item/stack/tile/bronze
+	custom_materials = list(/datum/material/bronze=MINERAL_MATERIAL_AMOUNT*5)
+	abstract_type = /obj/structure/statue/bronze
 
 /obj/structure/statue/bronze/marx
 	name = "\improper Karl Marx bust"
 	desc = "A bust depicting a certain 19th century economist. You get the feeling a specter is haunting the station."
 	icon_state = "marx"
 	art_type = /datum/component/art/rev
+
+
+/obj/item/chisel
+	name = "chisel"
+	desc = "breaking and making art since 4000 BC"
+	icon = 'icons/obj/statue.dmi'
+	icon_state = "chisel"
+	inhand_icon_state = "screwdriver_nuke"
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	flags_1 = CONDUCT_1
+	slot_flags = ITEM_SLOT_BELT
+	force = 5
+	w_class = WEIGHT_CLASS_TINY
+	throwforce = 5
+	throw_speed = 3
+	throw_range = 5
+	custom_materials = list(/datum/material/iron=75)
+	attack_verb_continuous = list("stabs")
+	attack_verb_simple = list("stab")
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	usesound = list('sound/items/screwdriver.ogg', 'sound/items/screwdriver2.ogg')
+	drop_sound = 'sound/items/handling/screwdriver_drop.ogg'
+	pickup_sound =  'sound/items/handling/screwdriver_pickup.ogg'
+	item_flags = EYE_STAB
+	sharpness = SHARP_POINTY
+
+	/// Block we're currently carving in
+	var/obj/structure/carving_block/prepared_block
+	/// The thing it will look like
+	var/atom/movable/current_target
+	/// Currently chosen preset statue type
+	var/current_preset_type
+	/// If tracked user moves we stop sculpting
+	var/mob/living/tracked_user
+	/// Currently sculpting
+	var/sculpting = FALSE
+/*
+ Hit the block to start
+ Point with the chisel at the target to choose what to sculpt or hit block to choose from preset statue types.
+ Hit block again to start sculpting.
+ Moving interrupts
+*/
+/obj/item/chisel/pre_attack(atom/A, mob/living/user, params)
+	. = ..()
+	if(sculpting)
+		return
+	if(istype(A,/obj/structure/carving_block))
+		if(A == prepared_block && (current_target || current_preset_type))
+			start_sculpting(user)
+		else if(!prepared_block)
+			set_block(A,user)
+		else if(A == prepared_block)
+			show_generic_statues_prompt(user)
+		return TRUE
+	else if(prepared_block) //We're aiming at something next to us with block prepared
+		set_target(A,user)
+		return TRUE
+
+// We aim at something distant.
+/obj/item/chisel/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!proximity_flag && !sculpting && prepared_block && ismovable(target))
+		set_target(target,user)
+
+/obj/item/chisel/proc/start_sculpting(user)
+	sculpting = TRUE
+	if(do_after(user,30 SECONDS, target = prepared_block))
```

Comment:
I mentioned this in discord as well but having a 30 second do after be interupted and you losing all your progress is a bit of a PITA, if you feel like it maybe you can make it so you need to carve a sculpture 10 times for 3 seconds to finish it, The user has to be a bit more active, and doesn't insta lose progress.